generators/server/templates/src/main/java/package/web/rest/ClientForwardController.java.ejs

```diff
@@ -31,7 +31,7 @@ public class ClientForwardController {
 <%_ if (communicationSpringWebsocket) { _%>
     @GetMapping(value = {"/{path:[^\\.]*}", "/{path:^(?!websocket).*}/**/{path:[^\\.]*}"})
 <%_ } else { _%>
-    @GetMapping(value = "/**/{path:[^\\.]*}")
+    @GetMapping(value = "/{path:[^\\.]*}")
```

Comment:
have you tested with base path?

Reply:
sorry, not sure to understand the question

Reply:
I think @vishal423 want you to test with base path, it means: localhost:8080/xxxx/

But honestly, it's just a pain to configure all the project, to make it work.

So for me, it's the correct fix and people who uses base path can test and propose an improvement here